apps/src/studio/studio.js

```diff
@@ -5916,6 +5974,7 @@ Studio.moveSingle = function (opts) {
   }
 
   if (skin.gridAlignedMovement) {
+    sprite.setActivity(constants.BEHAVIOR_GRID_ALIGNED);
```

Comment:
I _think_ it's the case that the else case expects `BEHAVIOR_STOP`, which is the default. If I'm correct then I could add that corresponding call for cleanliness, but if I'm not I have no idea what all could break